### PR TITLE
Checkbox layout on monitoring page on iPad

### LINKED ISSUE #265: Monitoring page checkbox layout problem on iPad
screen shot 2016-08-03 at 9 19 04 am

/cc @sg00dwin @jwforres

### PR DESCRIPTION
**Pull request by jwforres**
Fixes #265

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL, most of the lines are just whitespace changes

File: app/views/monitoring.html [hunk 1]
```diff
@@ -13,25 +13,27 @@
...
-              <div class="vertical-divider"></div>
-              <div class="data-toolbar-filter form-inline">
-                <div class="form-group filter-controls">
-                  <label for="events-filter" class="sr-only">Filter by name</label>
-                  <input type="search"
-                        placeholder="Filter by name"
-                        class="form-control"
-                        id="events-filter"
-                        ng-model="filters.text">
+            <div fit class="data-toolbar" style="justify-content: space-between;">
```
Comment (spadgett) [inline app/views/monitoring.html]: It's a little strange to set `justify-content` inline when nothing here in the markup makes it a flex container. It's only part the `data-toolbar` style. Maybe change this to `class="data-toolbar monitoring-toolbar"` and add the `justify-content` style to a `monitoring-toolbar` rule in `_data-toolbar.less`?

**Comment by spadgett** [comment]
One comment, otherwise LGTM

**Comment by jwforres** [comment]
[merge]

**Pull request merged**